Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -69,15 +73,26 @@ def audit
       files = ARGV.resolved_formulae.map(&:path)
     end
 
+    only_cops = ARGV.value("only-cops").to_s.split(",")
```

Comment:
What would you think about parsing values from `--only` instead? Feel free to decline, I'm not sure if I like it.